Samples/MediaCaptureSampleExtendedControl/App.xaml.cpp

```diff
@@ -0,0 +1,116 @@
+﻿//
+// App.xaml.cpp
+// Implementation of the App class.
+//
```

Comment:
copyright

Reply:
most files need this. Please check

---
In reply to: [373689950](https://github.com/microsoft/Windows-Camera/pull/11#discussion_r373689950) [](ancestors = 373689950)

Reply:
Fixed